### PR TITLE
Move CQL BP to Annex

### LINKED ISSUE #44: GeoSPARQL and Common Query language CQL
The common query language (CQL) (http://docs.opengeospatial.org/DRAFTS/19-079.html#simple-cql_filter-expression) is used in many geospatial web services (WFS) and in the currently in OGC API features to filter geospatial featurecollections.

In one of our research projects we developed a "semanticwfs" (https://github.com/i3mainz/semanticwfs), i.e. a WFS/OGC API Features compatible webservice with triple store backends which has to convert CQL filter expressions to (Geo)SPARQL internally.
Test Instance here: https://www.i3mainz.de/projekte/bkg/semanticwfs/

For this usecase (using a GeoSPARQL compliant triple store as a backend of an OGC API Features compatible web service) mappings from CQL to GeoSPARQL have to be defined.

If this is a usecase which is interesting to follow up we could define the new GeoSPARQL version keeping CQL compatibility in mind, as OGC API Features and WFS are still the main form of accessing geospatial data for the GIS community.

I am just putting this out there for discussion as I am not sure if it is something we should consider or if this is the right place.

### PR DESCRIPTION
**Pull request by nicholascar**
See it rendered: 

https://raw.githack.com/opengeospatial/ogc-geosparql/17fbd2f2f1d58d001174d56c2b4fced8d50f90c4/1.1/spec/00-GeoSPARQL.html

Closes #44

### CODE REVIEW

**Comment by nicholascar** [comment]
I agree that we _might_ have through that but I think, on reflection, it's just better to have it as an Annex as that just makes it easier to find. So that's why I put in the PR. Let's discuss!

**Comment by nicholascar** [comment]
I made a bunch of updates to the Annex/BP content here so even if it's ultimately not include as an Annex, it is updated.

**Review by situx — APPROVED** [review]

File: 1.1/spec/21-Annex-F.adoc [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+  ex:x rdfs:member ?item .
+  ?item rdfs:subClassOf* geo:Feature .
+}
+```
+
+GeoSPARQL's <<Class: geo:FeatureCollection, `geo:FeatureCollection`>> definition requires that <<Class: geo:Feature, `geo:Feature`>> instances are to be linked to the Collection by use of the ``rdf:member`` property. No inverse property is defined.
+
+[NOTE]
+====
+Some CQL-implementing systems, such as OGC API, have fixed notions of Feature Collections and require that Features be members of exactly on Feature Collection. There is no such restriction in GeoSPARQL: Features may be members of one or more Feature Collections.
```
Comment (mperry455) [inline 1.1/spec/21-Annex-F.adoc:26]: Typo "exactly on Feature Collection" -> "exactly one Feature Collection"

File: 1.1/spec/21-Annex-F.adoc [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+| beginTime ENDS 1969-07-16T13:32:00Z | N/A
+| beginTime MEETS 1969-07-16T13:32:00Z | N/A
+| beginTime METBY 1969-07-16T13:32:00Z | N/A
+| beginTime OVERLAPPEDBY 1969-07-16T13:32:00Z | N/A 
+| beginTime TCONTAINS 1969-07-16T13:32:00Z | N/A  
+| beginTime TEQUALS 1969-07-16T13:32:00Z | N/A 
+| beginTime TOVERLAPS 1969-07-16T13:32:00Z | N/A
+|===
+
+As noted above in Section <<F.2.1 Query Parameters, F.2.1 Query Parameters>>, temporal filtering of RDF data via SPARQL queries is possible with standard SPARQL functions to compare date values (`xsd:date`, `xsd:dateTime` and `xsd:dateTimeStamp` literals) and OWL TIME <<TIME>> may be used to asset temporal relations between objects.
```
Comment (mperry455) [inline 1.1/spec/21-Annex-F.adoc:156]: Typo: asset -> assert

File: 1.1/spec/21-Annex-F.adoc [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+CQL2 defines comparison predicates to compare two scalar expressions. A comparison predicate is converted to a triple pattern as shown in the following example. A SPARQL variable `?item` is assumed to represent the Feature Collection.
+
+|===
+| Comparison predicate  | Triple pattern | Comment
+
+| name="OGC" | `?item my:name "OGC"^^xsd:string` | Equality statements can be converted to a triple pattern
+| number=5 | ``?item my:number "5"^^xsd:integer`` | 
+| number>5 | ?item my:number ?number . FILTER(?number>5) | Arithmetic comparisons (<,>,>=,<=) are converted to filter expressions
+| number BETWEEN 5 AND 10 | `?item my:number ?number . FILTER(?number>=5 && ?number<=10)` | BETWEEN statements are converted to arithmetic expressions
+| name IN ("OGC","W3C") | `VALUES ?namevalues {"OGC" "W3C"} ?item my:name ?namevalues` | IN statements are expressed by VALUES statements
```
Comment (mperry455) [inline 1.1/spec/21-Annex-F.adoc:109]: Is there any reason not to convert CQL IN to SPARQL IN? VALUES also works. Just curious.

**Review by mperry455 — APPROVED** [review]

File: 1.1/spec/21-Annex-F.adoc (hunk 3 above)
Comment (nicholascar) [reply 1.1/spec/21-Annex-F.adoc]: I've flipped it to use `IN` and indicate `VALUES` can be used as I agree this is a more natural/direct mapping!

File: 1.1/spec/21-Annex-F.adoc (hunk 2 above)
Comment (nicholascar) [reply 1.1/spec/21-Annex-F.adoc]: thanks, done

File: 1.1/spec/21-Annex-F.adoc (hunk 1 above)
Comment (nicholascar) [reply 1.1/spec/21-Annex-F.adoc]: thanks, done

**Pull request merged**